server.js

```diff
@@ -242,6 +244,7 @@ app.use("/v1/sites", navigationRouter)
 app.use("/v1/sites", netlifyTomlRouter)
 
 app.use("/v2/auth", authV2Router.getRouter())
+app.use("/v2/sites", homepageV2Router.getRouter())
```

Comment:
just noticed - should we namespace the routers going forward? (don't need to fix here as out of scope)

Reply:
Yup that's a good point, I've raised an issue here https://github.com/isomerpages/isomercms-backend/issues/383